go.mod

```diff
@@ -39,3 +39,7 @@ require (
 	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace (
+    github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.9.0
```

Comment:
I'm confused why this has come up all of a sudden -- the Sirupsen -> sirupsen change happened many years ago and we don't have any `Sirupsen/logrus` import paths in the codebase...

If we do have any of the old import paths, can we just rewrite them?

Reply:
sorry, this was just noise from my previous attempt. the golang thing is all that's necessary.